types/v56/types.go

```diff
@@ -2019,7 +2019,7 @@ type NatRule struct {
 	// Elements
 	Description        string                 `xml:"Description,omitempty"`        // A description of the rule.
 	RuleType           string                 `xml:"RuleType,omitempty"`           // Type of NAT rule. One of: SNAT (source NAT), DNAT (destination NAT)
-	IsEnabled          bool                   `xml:"IsEnabled"`                    // Used to enable or disable the firewall rule. Default value is true.
+	IsEnabled          *bool                  `xml:"IsEnabled"`                    // Used to enable or disable the firewall rule. Default value is true.
```

Comment:
Is it still the case that `Default value is true`? I am thinking that we have no `omitempty`and it will send `false` by default.

Reply:
I need case when I don't send value. I added values where code used this field.

Reply:
Yep it is fine. I am just wondering if comment is correct - nothing else. Now it states - "Default value is true"

Reply:
updating comment